src/resources/views/crud/inc/datatables_logic.blade.php

```diff
@@ -50,7 +49,14 @@
 
     @endif
         if (saved_list_url && persistentUrl!=window.location.href) {
+
+            //we save the current alerts into local storage so we can read it later in alerts.blade.php
+            //after the user had been redirected by persistent table
+            localStorage.setItem('backpack_alerts', JSON.stringify(@json(\Alert::getMessages())));
```

Comment:
I'm wondering if it's better for the variable to be called `prologue_alerts` instead of `backpack_alerts`. Since that's the package name... though probably very few people are aware of that.

Personally I'm using the Prologue\Alerts package on the front-end too... so those would NOT technically be backpack alerts, but prologue alerts... Then again, it would require loading the JS to trigger the alerts. And normally I just point to the one in Backpack, or copy-paste it.

(1) What do you think would make more sense @pxpm ? `backpack_alerts` or `prologue_alerts`? 
(2) Also, let's get this right from day one - what's the naming convention others use for localStorage items? Or that _we_ use in other places? `snake_case`, `camelCase`, `PascalCase`?

Reply:
Indeed @tabacitu but you would not use datatables persistency feature in frontend, and notification will only endup in localstorage in case user is redirected from persistent table. If not we get the alerts from the usual alert bag, so I think `backpack_alerts` is the way to go since this functionality resides only inside admin panel.

2) We use `_` when storing datatables info.

Reply:
Excellent! Then `backpack_alerts` it is!